---
title: \textbf{Consequences of plant functional traits and plant-microbe interactions on species coexistence in heterogeneous landscapes}
author: 
- name: Gaurav Kandlikar
  affiliation: Dept. of Ecology & Evolutionary Biology- University of California, Los Angeles
- name: \newline Committee  
  affiliation: \hspace{11mm}\textbf{Dr. Nathan Kraft} (Chair), Dr. Jennifer Martiny, Dr. Lawren Sack, Dr. Felipe Zapata
abstract: "\\textbf{Abstract:}  
\\newline
How do interactions between organisms and their environment influence the dynamics of ecological communities? Ecologists have long understood that species performance can be impacted by both their response to the abiotic variation and by their interactions with other organisms. In my dissertation I aim to experimentally investigate how species coexistence is impacted by plant resource-use strategies (as summarized by functional traits) and species-specific interactions with soil microorganisms in an annual plant community in coastal California. This thesis will involve novel applications of recent advances in theoretical ecology to plant community ecology and integrate the historically disjunct fields of plant competition and plant-microbe interactions. 
\\newline
\\newline
A longstanding assumption has been that the coexistence of species with dissimilar functional traits is stabilized due to their distinct demographic responses to environmental heterogeneity [@Adler_2013]. For Chapter 1 I plan to test this assumption using recent advances in coexistence theory that quantify how variation in demographic responses to patchy landscapes can promote coexistence [@Chesson_2000]. Specifically, I propose to test whether functional trait differences are predictive of the similarity in the demographic responses of species to environmental gradients. 
\\newline
\\newline
Another striking observation from nature is that individuals within a species can have vastly different phenotypes. Despite considerable interest, the drivers of this variation ('intra-specific trait variation', or ITV) and its influence on species coexistence and community dynamics remain largely unclear [@Violle_2012; @Turcotte_2016]. For Chapter 2, I aim to investigate some of the biotic and abiotic drivers of ITV and ask how it might influence the outcome of competitive environments. 
\\newline
\\newline
Over the past two decades, a great deal of research has also highlighted the role of soil microorganisms in shaping terrestrial plant communities [@Bever_2015]. Microbes have been shown to promote plant performance in stressful conditions and have wreaked lasting ecological damage in other communities. In nature, such biotic interactions unfold concurrently with resource competition between plants; however in the literature, their effects are generally considered independently of one another [@van_der_Putten_2016]. For Chapter 3, I aim to demonstrate how the effects of resource competition and plant-microbe interactions can be jointly considered under the established coexistence framework of @Chesson_2008 via a critical review of existing plant-soil feedback literature."
date: "Draft- 6 September 2017"
output:
  bookdown::pdf_document2:
    template: ~/grtools/svm-latex-ms.tex 
    toc: false
fontfamily: mathpazo
fontsize: 11pt
header-includes: \usepackage{tabularx} \usepackage{float,booktabs,longtable,multirow,color,mathtools} \usepackage{graphicx,colortbl,xcolor,wrapfig}\usepackage{mathptmx,fixltx2e} \usepackage[font=small,skip=0pt]{caption}
csl: ecology.csl
bibliography: bibliography.bib
---

\setlength{\parindent}{0em}
\setlength{\parskip}{.5em}

\clearpage

## Background

Plant communities can house remarkable amounts of diversity. What are the fundamental processes that allow so many species to coexist with each other? Modern coexistence theory, emerging from the work of @Chesson_2000, frames species coexistence as an outcome of two types of differences between species. *Fitness differences* are differences between species that confer one species a density-independent competitive advantage over the other. For example, differential tolerance of a *generalist* predator can cause a fitness difference: in the absence of other processes, the more species that is less susceptible to the predator will outperform the more susceptible species. *Stabilizing niche differences* are differences between species that cause a species to limit its own population growth rate more than that of its competitor species. Strong interactions between species and *specialist* predators can act as stabilizing niche differences. Stabilizing niche differences must overcome any fitness differences between species to permit their coexistence (Figure \ref{fig:adler-fig} [@Chesson_2000; @Adler_2007]). In other words, coexistence is promoted by species differences that minimize fitness differences and those that increase stabilizing niche differences (although fitness equality alone is insufficient for coexistence without some stabilizing niche differences). 

\begin{wrapfigure}{l}{7cm}
  \includegraphics[width=7cm]{img/background-image.pdf}
  \caption{Coexistence is possible when stabilizing niche differences overcome fitness differences. Figure after Adler et al. (2007)  }
  \label{fig:adler-fig}
\end{wrapfigure}

**Trait based tests of coexistence**: What determines the magnitude of fitness and niche differences between species? One way to progress towards a general answer of this question is to understand the relationship between demographic dissimilarity and differences in species ecological strategy (as summarized by their functional traits [@Westoby_2002; @McGill_2006]). For example, rather than chasing the drivers of coexistence between pincushions (*Chaenactis* spp.) and goldfields (*Lasthenia* spp.) in California grasslands, we might consider coexistence as a function of how the two species build roots to extract soil resources. We could then extend this understanding to explain competitive dynamics of other species pairs with differing root structures. This generality is the promise of functional ecology [@McGill_2006]. 

As a first pass, ecologists extended the principle of limiting similarity [@Macarthur_1967] into the modern coexistence framework to hypothesize that functionally distinct species have high stabilizing niche differences and are more likely to coexist than functionally similar species. Recent advances in the field [@Mayfield_2010; @Kraft_2015] caution against such simplification. For example, @Kraft_2015 found that in an annual plant system, species differences in fruiting phenology are well correlated with both fitness and niche differences: species with later fruiting phenology tend to outcompete those with earlier phenology, but species pairs with dissimilar phenology have higher stabilizing niche differences than pairs with similar phenology. 

It is clear that to understand coexistence as a result of species ecological strategies, we need a better understanding of how species traits are linked to the demographic processes that belie coexistence. As spatial heterogeneity is thought to promote diversity in many plant communities by introducing stabilizing niche differences [@Chesson_1994], understanding the relationship between functional traits and species' demographic responses to environmental variation can provide a stronger foundation for future studies linking plant functional variation to coexistence in variable landscapes. 

**The microbial revolution**: A recent revolution in ecology has highlighted the ubiquity of microbes across ecosystems and the role they may play in generating and maintaining the diversity of larger organisms [@Fierer_2017]. In fact, the colonization of terrestrial habitats by plants may itself have been catalyzed by a mutualism with arbuscular fungi [@Humphreys_2010]. Evidence that soil microbes influence the dynamics of contemporary plant communities is mounting. For instance, @Mangan_2010 experimentally grew seedlings of six tropical tree species with microbes of various origins and showed plant species abundance may be driven by their susceptibility to soil microorganisms. @Laliberte_2015 have suggested that a tradeoff between plant phosphorus acquisition efficiency and susceptibility to pathogens may explain hyperdiversity in shrublands and tropical rainforests. 

Of course, interactions between plants and microbes play out simultaneously with the resource competitive dynamics that have been the focus of plant ecology for decades. For example,a specialist pathogen may introduce stabilizing niche differences to promote coexistence of a competitive inferior in a system with large fitness differences. Nevertheless, these two mechanisms are most often studied independently of one another [@van_der_Putten_2016], and the relative importance of microbial effects on plant coexistence are still unknown. Synthesizing these mechanisms under a unifying framework will be key to moving towards a better understanding of their combined effects on community dynamics. 

In my thesis, I propose to take a holistic view of plants from a perspective of their own physiology (as summarized by functional traits), their responses to environmental variation, and their interactions with microbes to ask three questions:

**Q1**: How do differences in functional traits relate to the differences in species' demographic response to environmental variation?    

**Q2**: How do plant traits vary across abiotic and biotic (competitive) gradients, and how does this affect the outcome of plant competition?   

**Q3**: How can we jointly consider the effects of plant-microbe interactions and resource competition on plant species coexistence? 

I will address the first two questions by performing experiments using the ~20 species from the annual plant community in Sedgwick Reserve, Santa Barbara County, California. These species occur across a heterogeneous landscape with diversity in topography and edaphic properties [@Gram_2004]. For Question 1, I have estimated the demographic response of annual plant species to environmental variation by planting seeds into 24 plots across the reserve. I plan to ask whether changes in species demography along environmental gradients are predicted by commonly measured plant functional traits. For Question 2, I again planted seeds across the reserve to test whether variation in functional traits within species (intra-specific trait variation or ITV) is linked to species demographic variation across the landscape, and whether ITV due to competitive neighborhoods can affect the outcome of competition. Finally, I propose to address Question 3 by showing how plant-microbe interactions can be considered in light of recent advances in coexistence theory that jointly consider effects of competition and natural enemies [@Chesson_2008]. I plan to conduct a critical review of the plant-soil feedback literature and to extend @Chesson_2008's framework to include the consequences of mutualistic interactions. My thesis will combine recent advances in coexistence theory with empirical advances in plant and microbial ecology to contribute towards a holistic and mechanistic understanding of plant diversity maintenance.

\clearpage 

## Chapter 1: How do functional traits relate to species responses to variable environments?

*Additional collaborators: Andy Kleinhesselink (UCLA); Jonathan Levine (ETH-Zurich); Emily Curd (UCLA/Cal eDNA)*

\bigskip
\indent
A foundational principle of trait-based ecology is that traits are functional to the extent that they influence population demography by directly influencing growth, survival, and reproduction [@Violle_2007]. As variation in demographic properties is at the heart of species coexistence [@Chesson_2000], understanding the links between species traits and their demography is a critical step in using traits to understand coexistence. Although ecologists have long understood that trait differences can reduce the niche overlap between species to promote coexistence [@Macarthur_1967], recent advances have shown that trait differences can also promote competitive exclusion [@Adler_2013; @Kraft_2015]. [*I think this transition is awkward right now*] . As nature is often strikingly patchy in space and as species often have clumped distributions on landscapes, spatial variation is thought to be an important driver of coexistence in plant communities [@Chesson_2000]. Therefore, traits likely influence species coexistence by their effect on species' demographic responses to variable environments.  

Despite being fundamental to interpreting patterns of trait variation in nature, the links between trait differences and species' demographic responses remain largely unresolved [@Shipley_2016, but see @Adler_2014 and @Wright_2010 for counter-examples]. Many studies have shown non-random associations between trait values and the environment [@Cornwell_2009; @Fortunel_2014]. However, it is difficult to interpret these results in the context of the demographic drivers of coexistence. For example, one limitation of these studies is that observed abundances are influenced not only by environmental variation but also by variation in competitive neighborhoods between sites [@Adler_2013]. Indeed, the most abundant species at a site need not have the traits that confer the highest fitness at that site [@Fox_2012]. Therefore, isolating the impact of environment while keeping the competitive environment constant is a critical step in building stronger links between functional trait differences and variation in species responses to variable environments [@Adler_2013]. Interpreting these results in light of ecological first principles regarding trait-environment relationships can give generality to our findings.

In addition to the competitive neighborhood, the interactions between plants and microbes may also change across space to influence plant performance. Turnover of soil microbes can have important effects on plant responses to abiotic environmental variation [@Keymer_2017]. For example, @Lau_2012 found that the fruit production of *Brassica rapa* was less impacted by a drought treatment when the plants were grown with a drought-adapted soil microbial community than when grown with a soil microbial community that had not been previously exposed to drought. Therefore, the soil microbial community may function as an additional environmental variable that to date has received relatively little attention from plant ecologists. Recent advances in sequencing technology have made it feasible to characterize variation in soil microbial turnover, and we can begin to assess their role in driving plant environmental preferences.

To address these gaps, I propose to use experimental data to estimate the contribution of trait differences to species-specific demographic responses to environmental variation, including turnover in soil microbial communities, for the first chapter of my dissertation. 

### Hypotheses

\textbf{H\textsubscript{0}}: Species may respond identically to environmental variation, irrespective of functional traits. We may expect this pattern if the environmental gradient spans a single stress gradient axis such that all species perform poorly at some sites and well at others (Figure \ref{fig:ch1-hypotheses}a). In this case, observed patterns of species turnover across the landscape at Sedgwick may be driven by poor dispersal.   
\textbf{H\textsubscript{1}}: Species may have idiosyncratic responses to the environmental gradient, but the response may be unrelated to functional traits. There is potential for environmental variation to promote species coexistence, but the stability of coexistence would be uncorrelated to species' functional similarity (Figure \ref{fig:ch1-hypotheses}a).  
\textbf{H\textsubscript{2}}: Variation in responses of species to environmental gradients may be explained by their functional traits. This pattern might arise if the functional traits are indicative of species' environmental preference along the gradient sampled (Figure \ref{fig:ch1-hypotheses}a).  
\textbf{H\textsubscript{3}}: Similarity in some traits may correlate with similarity in response to certain environmental variables. For example, drought tolerance may predict demographic responses to an aridity gradient but not to a light gradient, and seed size may predict responses to a soil texture gradient but not an aridity gradient (Figure \ref{fig:ch1-hypotheses}b). 


\begin{figure}[h!]
  \includegraphics[width=\textwidth]{img/ch01-hypotheses.pdf}
  \caption{Potential ways in which trait differences may correlate with differences in species' demographic response. Note: I don't know if this is the best way to present, or if i should revert to a simpler structure like I had for ESA}
  \label{fig:ch1-hypotheses}
\end{figure}


### Methods

\begin{wraptable}{r}{5cm}
```{r species, echo = F, results='asis', warning=FALSE, echo = F}
library(magrittr); library(knitr); library(kableExtra)
species <- c("Agoseris heterophylla","Centaurea melitensis" ,"Chaenactis glabriuscula","Hemizonia congesta" ,"Lasthenia californica", "Micropus californicus", 
             "Amsinckia menziesii", 
             "Clarkia bottae","Clarkia purpurea", 
             "Euphorbia peplus", 
             "Medicago polymorpha", "Lotus wrangeliensis", 
             "Bromus madritensis","Hordeum murinum", "Vulpia microstachys", 
             "Salvia columbariae", 
             "Plantago erecta")
# species <- paste0("\\textit{", species, "}")
# species <- paste0("**", species, "**")
families <- c("Asteraceae", rep(" ", 5), "Boraginaceae", "Onagraceae", " ", "Euphorbiaceae", "Fabaceae"," ", "Poaceae", rep(" ", 2), "Lamiaceae", "Plantaginaceae")

to_print <- cbind(Family = families, Species = species)
kable(to_print, format = "latex" , longtable = TRUE, caption = "Species list for demography experiment", booktabs = T, linesep = c(rep("",5, "\\addlinespace")))  %>% kableExtra::column_spec(2, italic = T) %>% kable_styling(font_size = 8)
```
\end{wraptable}


**Demographic variation**: In 2015-2016, we planted seeds of 17 annual plant species (Table 1) at 24 plots in a setup that allowed me to parameterize a simplified annual plant population growth model. In this model, the population size of a species $j$ at a site $x$ grows according to it seed bank survival rate ($s_j$) and its intrinsic germination and seed production response at each site ($g_{j,x}$ and $\lambda_{j,x}$ respectively), but the rate of growth is reduced by how much competition it faces at each site ($\eta_x$) and how sensitive it is to competitors ($r_{j,x}$): 
$$\frac{N_{t+1,j,x}}{N_{t,j,x}} = (1-g_{j,x})*s_j +\frac{g_{j,x}*\lambda_{j,x}}{1+r_{j,x}*\eta_x}$$
In this framework, environmental response can be parameterized in three ways: germination rate, the per-capita seed production in the absence of competitors, and a composite index that yields the expected population growth rate in the absence of competitors.   


**Trait variation**: A set of plant, seed, root, and whole-plant functional traits were measured for twelve of the seventeen species in this experiment by @Kraft_2015 (Table 2). I have measured the same set of traits for the five remaining species and have additionally measured leaf osmotic potential [@Bartlett_2012], which is a strong predictor of leaf drought tolerance .

**Abiotic environmental variation**: We characterized various environmental properties at each of our experimental sites during the 2015-16 growing season. Soil was sent to Western A&L Labs for chemical and physical analysis. Air temperature at each of the plots was measured with temperature data loggers (iButtons) that were placed inside PVC tubing to shield from solar radiation, and gravimetric water content was measured for twice for each plot - once each in the peak growing season for the early and later phenology species. In analyses, I will use both raw environmental variables and NMDS scores to characterize the environment.

\begin{wraptable}{l}{5cm}
```{r traits, echo = F, results='asis', warning=FALSE}
traits <- c("Leaf size", "Specific Leaf Area (SLA)", "Leaf Dry Matter Content (LDMC)", "Osmotic Potential", "Leaf Nitrogen Concentration", "Seed Mass", "Specific Root Length (SRL)","Rooting Depth", "Max. Height", "Canopy Shape Index", "Fruiting Phenology", "Carbon Isotope Ratio")
category <- c("Leaf", rep(" ", 4), "Seed", "Root", "Plant", rep(" ", 5))

to_print <- cbind(Trait = traits)
kable(to_print, format = "latex" , longtable = TRUE, caption = "List of functional traits", booktabs = T, linesep = c(rep("",5, "\\addlinespace"))) %>% kable_styling(font_size = 8)
```

\end{wraptable}
**Biotic environmental variation**: We characterized the variation in soil microbiota among our plots by using metagenomic sequencing to profile the diversity of bacteria, fungi, and small eukaryotes. Soil was pooled from three equally spaced points within each experimental plot into a single sterilize 15 mL tube and stored at -80$^{\circ}$C until further processing. Total DNA was extracted using the MoBio Power Soil Extraction kit and PCRs were conducted using 16s rRNA primers for bacteria, fungal ITS primers, and 18s rRNA primers for small eukaryotes. Pooled samples were sequenced on the Illumina MiSeq platform at the UC-Berkeley sequencing facilities. Sequence data will be used to ordinate sites according to their soil microbial communities in multidimensional space.    

\bigskip 

**Analyses**: I propose to assess the influence of functional traits on the demographic responses of species to environmental gradients with a generalized linear mixed modeling (GLMM) framework modified from [@Jamil_2012]. In this framework, the species performance across a landscape ($p_{i,j}$) is modeled as a function of species traits $z$, environmental properties $x$, and crucially, an interaction term between the two: 
\[  
p_{i,j} \sim \overbrace{a_0}^{\substack{species\\ effect}}+\underbrace{a_1z_j}_{\substack{trait\\ effect}}+\overbrace{b_0x_i}^{\substack{environment\\ effect}}+\underbrace{b_1z_jx_i}_{interaction}+\overbrace{\epsilon_{\alpha j} + \epsilon_{\beta i} + \gamma_i}^{error\ terms}
\]

The species effect term $a_0$ is a fixed effect that can be thought of as capturing unmeasured species traits, and the random effect term $\gamma_i$ represents unmeasured environmental variation. The contribution of trait differences to interspecific variation in environmental response (i.e. the slope parameter $\beta_1$) can be quantified as a function of the variance explained ($\hat\sigma_b$) by a model that includes a trait:environment interactiom term and one that lacks it:

$$C_\beta = 1-\frac{\hat\sigma^2_{\beta,\ no\ interaction\ term}}{\hat\sigma^2_{\beta,\ with\ interaction\ term}}$$


### Preliminary results, implications, and further analyses
Preliminary analyses show that most of the species pairs had uncoupled demographic responses- in other words, most species pairs did not respond in the same way to the multivariate changes in environment across sites (Figure \ref{fig:ch1-results}a), which suggests against **H\textsubscript{0}**. Thus, variable demographic responses of species to environmental heterogeneity may contribute to their (Figure \ref{fig:ch1-results}b). The environmental variation among our sites stems primarily from differences in soil [Ca\textsuperscript{2+}]:[Mg\textsuperscript{2+}] ratio and in soil texture and depth (Figure \ref{fig:ch1-results}b). The species in this experiment differed primarily in leaf economics spectrum traits on the first axis of variation and on whole plant traits on the second axis of variation. Preliminary analyses conducted with simple linear regressions hint at some trait differences that may explain variation in species responses to environment. For example, four out of the five smallest-seeded species in this experiment perform better at sites with higher values of soil NMDS 2 (shallower and sandier soils), whereas all other species prefer deeper soils with higher clay content. 


Modeling variation in demographic response to environment as a function of traits in this framework above will address a fundamental gap of trait-based plant ecology: the assumption that functional traits have direct and predictive relationships with measurable environmental gradients [@Adler_2013;@Shipley_2016]. Due to strong separation of sites along axes of soil chemistry and physical structure, we may find that root and seed traits that directly link to species' belowground ecological strategies are more relevant to explaining variation in species response in our system than traits that primarily capture aboveground strategies. Alternatively, physiological traits with a clear link to an environmental axis (e.g. leaf osmotic potential to water availability) may show stronger links to plant performance than integrative traits like SLA that are related to various environmental axes. An important step will be to conduct a power analysis to assess the extent to which the potentially subtle but important influences of individual traits (as might be expected for some of the integrative functional traits measured for this study) can be recovered given the probabilistic nature of many demographic models (Snyder & Ellner, *in prep*). A number of studies have also found phylogenetic structure in plant communities; therefore, incorporating evolutionary non-independence of our samples by using phylogenetically informed GLMMs [@Ives_2011] is another promising next step. I plan to interpret all results in light of ecological first-principles and in the framework of modern coexistence theory to yield generality to these findings. Armed with a more robust understanding of the link between traits and demography, trait-based plant ecology can move towards its potential to be a predictive science in a rapidly changing world [@McGill_2006]

\begin{figure}[h!]
  \includegraphics[width=\textwidth]{img/ch1-fig2.png}
  \caption{\textbf{A.} Species tended to show decoupled demographic responses (all pairwise correlations in species responses <1. \textbf{B.} Soil chemical and physical properties were the primary axes of variation among sites.}
  \label{fig:ch1-results}
\end{figure}

## Chapter 2: The nature and consequences of intra-specific trait variation

Individuals within a species can have vastly different phenotypes- indeed, genetic variation within populations is the fundamental basis for natural selection. Intra-specific variation (ITV) in plant functional traits can be quite high: a recent meta-analysis [@Siefert_2015] showed that 25% of functional trait variation within communities is driven by ITV. Variation in functional traits within species has many causes. Among other drivers, traits can vary due to genotype [@Brouillette_2013], ontogeny [@Spasojevic_2014], fine-scale environmental variation (e.g. between shaded vs. sun-exposed leaves) [@Nobel_1976], regional environmental variation [@Cornwell_2009], and local competitive neighborhoods [@Turcotte_2016]. 

How important is this ITV to understanding the structure of natural communities? Although community dynamics fundamentally emerge from interactions between individuals, ecologists tend to rely on species averages to study the drivers of species coexistence, leaving the question of ITV impacts on coexistence largely unexplored [@Violle_2012; @Turcotte_2016]. The ability to infer community-wide dynamics from species-mean trait values measured on a small number of individuals per species is at the core of the generalizability of trait-based ecology [@McGill_2006; @Shipley_2016]; therefore, it is important to understand how ITV may affect these processes. *I think this part still needs work. Will think on it.*

Whether ITV promotes or hinders coeixstence, and whether observed levels of ITV affect our predictions of the outcome of competitions, still remains unknown. One line of reasoning suggests that variation among individuals of a species may decrease species-level fitness differences [@Fridley_2007] or promote stabilizing niche differences [@Clark_2010] and thus make coexistence more likely. On the other hand, @Hart_2016 demonstrated with a theoretical model incorporating viability in demographic rates that ITV is more likely to destabilize coexistence due to non-linear averaging. However, an important limitation of this model was that it assumed a well-mixed population with no structure in ITV- in other words, the demographic traits of an individual were randomly drawn from a population distribution and were not a function of their local environment. 

ITV is unlikely to be random in nature, and we need a better understanding of how abiotic and biotic environments shape ITV to better understand its potential effects on coexistence [@Turcotte_2016]. However, mapping the structure of ITV alone is not sufficient to understanding its influence on coexistence. An understanding of how functional traits relate to the demographic drivers of coexistence is needed to incorporate ITV into coexistence predictions. As @Kraft_2015 have shown how functional traits relate to fitness differences and stabilizing niche differences in the Sedgwick annual plant system, this community is an ideal one in which to ask these questions: 

**Q1**: What are the patterns of ITV in response to a strict environmental gradient (i.e. one across which competitive neighborhoods are not changing)?     
**Q2**: How do traits change in response to competitive background, and how can this affect the outcome of competition?     

### Hypotheses  

\textbf{Q1, H\textsubscript{0}}: Species functional traits may not change across strict environmental gradients. In this case, observed patterns of community-wide trait turnover across gradients may be driven by species turnover or due to trait responses to competitive neighborhoods.    
\textbf{Q1, H\textsubscript{1}}: Functional traits may shift in the same way across species in response to environmental gradients (e.g. trait values may increase across the gradient for all species).    
\textbf{Q1, H\textsubscript{2}}: Traits may shift across environmental gradients, but the slope and magnitude of trait shifts may differ between species.    
\textbf{Q1, H\textsubscript{2a}}: Species that are less demographically sensitive to the environmental variation (i.e. species with high population growth rates at all sites) may show higher ITV than species that are more sensitive to the environment.   

\textbf{Q2, H\textsubscript{0}}: Species functional traits may not shift in response to competitive backgrounds.    
\textbf{Q2, H\textsubscript{1}}: Functional traits may shift in the same way across species in response to competitive backgrounds (e.g. trait values may be higher in competition than when grown alone, irrespective of species).    
\textbf{Q2, H\textsubscript{2}}: Traits may shift across competitive backgrounds, but the slope and magnitude of trait shifts may differ between species. In this case, ITV might have the potential to shift the outcome of competition by influencing the magnitude of niche and fitness differences between species.    
\textbf{Q2, H\textsubscript{2a}}: Traits of competitive inferiors may respond more strongly than those of competitive superiors in a way that decreases fitness differences or increases niche differences [@Kraft_2015]; alternatively, trait changes may shift in ways that reinforce the dominance of the competitive superior. 


### Methods  

**Trait sampling**: In 2016-2017 we planted seeds of 20 species in the same locations we had used for our demography experiment in Chapter 1 Seeds were planted 15 cm away from one another to ensure that plants were growing with a very low density of competition so that we could estimate the impact of environment alone on traits for **Q1**. Additionally, at one site, we planted all species in a high competitive density to estimate the effects of competitive neighborhoods on traits for **Q2**. We measured leaf size, SLA, and LDMC on three leaves from each individual. We additionally measured plant height and canopy shape index on the same individuals and stored root tissues in ethanol to measure specific root length (SRL). In 2017-2018, I plan to sample these same traits on individuals growing alone, with conspecific competitors, and with heterospecific competitors to test if trait response to competitors depend on the identity of the competitor species [@Turcotte_2016]. (*Nathan, it might be good to talk about this- I think it'd be nice to round out this part of the story if there is a way to easily sample individuals from Chips Ahoi, because Turcotte and Levine highlight it as the key ingredient that's always missing*)

**Analysis**: To answer **Q1**, I propose to perform GLMM analyses similar to those being used in Chapter 1 to describe trait shifts in response to environmental gradients. Specifically, I propose model the traits of species $i$ at site $j$ as 

$$trait_{i,j} \sim \overbrace{a}^{\substack{trait\\ mean}} + \underbrace{b_0x_i}_{\substack{species\\ effect}} + \overbrace{c_0z_j}^{\substack{environment\\ effect}} + 
\underbrace{c_1x_iz_j}_{interaction} + \overbrace{\epsilon_{\alpha j} + \epsilon_{\beta i} + \gamma_i}^{\substack{error\\  terms}}$$

The strength of the environment's effect on trait values is captured by the term $c_0z_j$, and differences in species response to the environment are modeled by the interaction term $c_1x_iz_j$. 

As a preliminary test of whether functional traits change with competitive background and whether such a change is species-specific (**Q2**), we performed a two-factor ANOVA testing for the influence of species identity, competition background, and their interaction on the three leaf traits (leaf area, SLA, LDMC). If species have idiosyncratic responses to competition, then trait shifts in response to competition might have potential to change competitive outcomes. Finally, I will use the trait-based models of fitness and niche differences from @Kraft_2015 to test whether the predicted outcome of coexistence changes after incorporating ITV. [*I think this last part needs a bit more meat to it, need to work on this*]

### Preliminary results, future analyses, and implications

In total, we measured traits on >800 individuals for this study, with an average of 40 individuals per species. In our preliminary analysis for **Q2**, we found species- and trait-specific consequences of competitor density on traits. Specifically, we found that plants growing in dense competition tended to have higher SLA and larger leaves than those of the same species grown at low-density (p < .0001). Competitive density had no clear effect on LDMC. There was a significant interaction between species and competitive background for all three traits, indicating species-specific responses to competition (Figure \ref{fig:ch2-results}). Low germination of some species resulted in an unbalanced dataset; therefore, it will be important to perform further nonparametric tests (e.g. REML F-tests [@Bolker_2009]) to verify the robustness of these results. 

These preliminary results suggest that shifts in traits in response to competitors have the potential to change the predicted outcome of competitive interactions. For example, species with more resource conservative leaf traits (e.g. low SLA, high LDMC) tend to be competitive superiors in this system [@Kraft_2015]; therefore, idiosyncratic responses of these traits to competition may serve to either increase or decrease fitness differences. As a first test of this possibility, I will use the trait-based models of species interactions from @Kraft_2015 to ask whether accounting for all of the observed intraspecific variation significantly changes the magnitude of fitness and niche differences. This approach would be similar to that of @Hart_2016 in assuming a well-mixed community with no structure in ITV. I will then perform more targeted analyses that ask if ITV in response to competition specifically changes the predictions of coexistence.

Although functional traits can vary significantly within species, the consequences of this variation on species coexistence are unclear. By quantifying the impact of ITV in functional traits to variation in species demography, this study will begin to address an important gap between the theoretical work that focuses on ITV in demographic variation and the large body of empirical work that has documented ITV in functional traits.

\begin{figure}[h!]
  \includegraphics[width=\textwidth]{img/ch2-results.png}
  \caption{Two-way ANOVAs suggest that leaf functional traits have idiosyncratic responses to competitive backgrounds across species.}
  \label{fig:ch2-results}
\end{figure}


## Chapter 3: Considering the consequences of soil microbial communities on plant species dynamics under the modern coexistence framework

Microbes are ubiquitous in nature and can have dramatic impacts on the performance of larger organisms. @Redman_2002 found that when colonized by endophytic fungi in the genus *Curvularia*, the Panic grass *Dichanthelium lanuginosum* could withstand sustained soil temperatures greater than 50$^{\circ}$C. This interaction is critical to understanding the persistence of the grass in geothermal soils where few other plants occur. Conversely, microbes can severely harm plant populations. For instance, the oomycete *Phytophthora ramorum* caused dramatic mortality of tanoak (*Lithocarpus densiflorus*) and coast live oak (*Quercus agrifolia*) in coastal California and Oregon in the late 1990s [@Rizzo_2005]. This continues to have profound impacts on the community and ecosystem dynamics in those systems [@Cobb_2013]. Theoretical and empirical advances in the last two decades have highlighted that microbes can also have subtler mechanisms for effecting profound changes in plant communities. For example, @van_der_Putten_1993 suggested that plant succession in foredune communities is driven by the promotion of late-successional plant species in soils that harbor a microbial community cultivated by early-successional species. In a generalized theoretical framework that has stimulated a great deal of research, @Bever_1997 showed how feedback loops between plants and soil microbes ("plant-soil feedbacks", or PSFs) could permit the coexistence of competing plant species. In short, plant-microbe interactions have the potential to dramatically influence the outcomes of plant species interactions and community assembly. 

But how important are these effects in comparison to the competitive effects that are the subject of traditional coexistence and community assembly theory? We cannot answer this question without a framework that simultaneously considers the effects of both mechanisms. For the third chapter of my dissertation, I propose to demonstrate that the effects of plant-microbe interactions can be considered jointly with the effects of resource competition under the competition-predation framework of @Chesson_2008 and critically review the empirical work conducted under the plant-soil feedback framework of @Bever_1997 in this new context. Fundamental to this reframing is the fact that microbes can influence plants in ways that are independent of any feedbacks (i.e. some microbial contexts always benefit or always harm plant species, regardless of plant species density) to influence fitness differences, and via feedbacks (i.e. the impacts of some microbes on plant species may change with plant species density) to influence stabilizing niche differences. Here I briefly review the state of PSF research, summarize @Chesson_2008's coexistence framework, and begin to show how the former can be integrated into the latter. 

**State of plant-soil feedback literature**: Plants can alter both biotic and abiotic characteristics of the soil. Plants that later grow in those soils will be responding to the changes- this cyclical relationship is the basis of plant-soil feedbacks (PSFs). @Bever_1997 showed that if the effects of the soil community cultivated by a plant species $A$ are more negative to the population growth rate of species $A$ than the growth rate of plant species $B$, and vice-versa (i.e. more negative intra-specific effects than inter-specific effects), then PSFs can promote species coexistence. Alternatively, positive PSFs can drive competitive exclusion by preferentially promoting the performance of the abundant species. This framework has inspired many studies (a Web of Science search for titles containing "plant-soil feedback*" yielded 171 results overall and 73 in 2015-2017 alone) which have documented a diversity of positive, negative, and neutral effects of PSFs. 

Despite such extensive experimentation, it remains difficult to predict the importance of PSFs: their impacts seem to be highly species- and environment-specific [@Casper_2007; @van_der_Putten_2016]. Recently, @Smith_Ramesh_2017 predicted that PSFs are less important for coexistence in higher nutrient sites where plants can have high immunity and produce more defence compounds, whereas @van_der_Putten_2016 predict that the higher performance of pathogens in higher nutrient sites will increase the importance of negative PSFs. Additionally, most experimental work is conducted under highly controlled greenhouse settings, and it is difficult to translate these results into the importance of PSFs in nature [@van_der_Putten_2016]. Finally, an important limitation of this framework from the perspective of coexistence theory is that it focuses solely on one set of processes that might promote coexistence (namely, stabilizing niche differences) but fails to incorporate potential effects of microbes on fitness differences (for example, by acting as a generalist enemy that drives fitness differences). Reframing plant-soil interactions in modern coexistence theory may highlight heretofore overlooked parameters and suggest avenues of research to start building generalities. 

**Overview of competition/predation framework**: That shared natural enemies can change the outcome of strict resource competition has been long understood in ecology as "apparent competition" [@Holt_1977]. @Chesson_2008 integrated apparent competition into modern coexistence theory to show how natural enemies can affect coexistence in much the same way as resource competition. If natural enemies are specialists, then intra-specific feedback loops are stronger than inter-specific feedback loops and coexistence is promoted via lower niche overlap. On the other hand, physiological tradeoffs between natural enemy defense and rates of reproduction can promote coexistence via equalizing fitness differences [@Chesson_2012]. Starting from a system that modeled the dynamics of enemies, prey, and resources, @Chesson_2008 derived equations to quantify niche overlap ($\rho$) and fitness ($\kappa_j$) as a joint function of resource competition and natural enemies: 

$$\rho = \frac{ \sum_{l} \frac{c_{jl}V_{l}c_{kl}}{r_l^R\alpha_I^R} + \boxed{\sum_{m} \frac{a_{jm}wa_{km}}{r_m^P\alpha_m^P}}}{\sqrt{\left( \sum_l\frac{c_{jl}^2v_l}{r_l^R\alpha_l^R} +  \boxed{\sum_l\frac{a_{jm}^2w}{r_m^P\alpha_m^P}}\right)\left( \sum_l\frac{c_{kl}^2v_l}{r_l^R\alpha_l^R} +  \boxed{\sum_l\frac{a_{km}^2w}{r_m^P\alpha_m^P}}\right)}}$$


$$\kappa_j = \frac{\sum_l \frac{c_{jl}v_l}{\alpha_l^R}-\boxed{\sum_m\frac{a_{jm}}{\alpha_m^P}} - \mu_j}{\sqrt{\left(\sum_l \frac{c_{jl}^2v_l}{r_l^R\alpha_l^R} + \boxed{\sum_m \frac{a_{jm}^2w}{r_m^P\alpha_m^P}} \right)}}$$

The parameters are as follows: $c_{jl}$ represents the consumption rate of resource $l$ by species $j$; $v_l$ is the unit value of resource $l$, $\alpha_l^R$ is the reciprocal of resource carrying capacity, $r_l^R$ is the rate or resource replacement, $a_jm$ is the attack rate of enemy $m$ on species $j$, $w$ is the unit value of the focal species to the enemy, $\alpha_m^P$ is the reciprocal of enemy carrying capacity, and $r_m^P$ is the enemy's intrinsic population growth rate. All boxed terms indicate the effects of natural enemies, whereas all unboxed terms indicate the effects of resource competition (save $\mu_j$, which represents maintenance costs of species $j$). **The key takeaway** is that in this framework, niche differences and fitness differences are both simultaneously influenced by resource competition as well as natural enemies. 

### Preliminary results, further analyses, and implications 

I have implemented the model demographic model from @Chesson_2008 as an `R` webapp (https://github.com/gauravsk/chesson-2008-exploration). Analysis of @Chesson_2008's model shows that the net impacts of plant-microbe feedbacks can be influenced by several parameters. For example, consider the consequences of adjusting two parameters in the model: 1) the rate of resource regeneration ($r_l^R$ and $r_g^R$), and 2) the degree to which enemy populations can grow independently of the focal species (i.e. due to predation on other species in the system) ($r_m^P$ and $r_n^P$). The relative importance of PSFs is higher in more fertile sites (higher resource replacement rates $r^R$) and when microbes interact weakly with species outside of the focal group (low $r^P$, Table \ref{tab:ch3-table}). The insight regarding the increased importance of PSFs in sites with high resource replacement rates is in contradiction with the prediction of @Smith_Ramesh_2017, who suggest that PSFs should be less relevant in more fertile sites but fail to consider the **relative importance** of PSFs and resource competition. However, it will be important to analyse both the steady-state and transient dynamics (e.g. time to competitive exclusion) of the model, as many empirical studies focus on the effects of varying state variables (e.g. total resources level at time $t$), rather than model parameters (e.g. resource replacement rate).

I plan to continue exploring the behavior of this model and find generalities to help interpret the many contradictory ideas regarding PSFs and their role in shaping plant communities. To extend the framework, I plan to modify the model to incorporate potential mutualistic impacts of microbes on plant population growth. This can be done by setting the attack rate $a_jm$ to be a negative number. After exploring the dyncamics of the model, I propose to conduct a thorough review of the literature since @Bever_1997 and interpret the results in light of the joint competition-natural enemies model. This review will highlight areas that have received little attention but are important to understanding the role of PSFs in nature. Finally, I will use this framework to propose future experimental work that rigorously tests the role of PSFs relative to that of resource competition in natural systems [*Maybe I should write more here- maybe propose a clean greenhouse experiment*]. For example, in 2016 I conducted a growth chamber experiment to test for whether fitness differences (as estimated by germination rate and vegetative biomass) were impacted by soil microbial context across five sites with varying soil chemistry. Such experiments can be used to test specific predictions regarding the *net effects* of resource competition and microbial dynamics on plant communities. Reframing PSFs under the modern coexistence framework will help the field meet its new frontier of understanding variation in the relative importance of PSFs in nature.


```{r ch3-table, results = 'asis', echo = F, eval = T}
tab <- read.csv("~/grad/quals/proposal/ch3-table.csv")
colnames(tab) <- c("$a_{jm}$","$a_{jn}$","$a_{km}$","$a_{kn}$","$r_l$", "$r_g$" ,"$r_m$"," $r_n$" ,"$\\rho_{net}$"," $\\rho_{competition}$"," $\\rho_{enemies}$","$\\frac{\\kappa_j}{\\kappa_k}$" ,"coexist?")
print(xtable::xtable(tab, align = rep("c", 14), label = "tab:ch3-table", 
                     caption = "Contribution of niche differentiation due to natural enemies ($\\rho_{enemies}$) to net niche differentiation ($\\rho_{net}$) increases with resource replacement rates (higher $r_l$ and $r_g$) and when enemies depend strongly on the focal species (lower $r_m$ and $r_n$) 
                     \\newline
                     The values of other parameters were as follows: $c_{jl}$ = .04,$c_{jg}$ = .02, $c_{kl}$ = .02,$c_{kg}$ = .02,  $v_g$ = $v_l$ = .2, $u_j$ = $u_k$ = .1, $W$ = .01, $\\alpha_g$ =$\\alpha_l$ = .05, $\\alpha_m$ = $\\alpha_n$ = .1"),sanitize.text.function=function(x){x}, comment = FALSE, include.rownames=FALSE, digits = 4)
```



## Timeline

|**Year**|  **Season** |  **Plan** |
|:-----:|------|--------------------------------------------------------|
| 2017 | Fall-Winter  | Analysis and writeup for Chapter 1; Literature reviews for Chapter 3   |
| 2018 | Spring-Summer| Writeup and submission for Chapter 1; Additional data collection for Chapter 2; Literature reviews & modeling for Chapter 3|
| 2018 | Fall-Winter  | Analysis for Chapter 2, Modeling & analysis for Chapter 3 |
| 2019 | Spring-Summer| Writeup and Analysis for Chapter 2, Writeup for Chapter 3|
| 2019 | Fall-Winter  | Writeup for Chapter 2; Submission for Chapter 3|
| 2020 | Spring| Submission for Chapter 2|

\clearpage

## References

