---
title: \textbf{Consequences of plant functional traits and plant-microbe interactions on species coexistence in heterogeneous landscapes}
author: 
- name: Gaurav Kandlikar
  affiliation: Dept. of Ecology & Evolutionary Biology- University of California, Los Angeles
- name: \newline Committee  
  affiliation: \hspace{11mm}\textbf{Dr. Nathan Kraft} (Chair), Dr. Jennifer Martiny, Dr. Lawren Sack, Dr. Felipe Zapata
abstract: "\\textbf{Abstract:}  
\\newline
How do interactions between organisms and their environment influence the dynamics of ecological communities? Ecologists have long understood that species performance can be impacted by both their response to the abiotic variation and by their interactions with other organisms. In my dissertation I aim to experimentally investigate how species coexistence is impacted by plant resource use strategies (as summarized by functional traits) and species-specific interactions with soil microorganisms in an annual plant community in coastal California. A longstanding assumption has been that the coexistence of species with dissimilar functional traits is stabilized due to their distinct responses to environmental heterogeneity [@Adler_2013]. For Chapter 1 I plan to test this assumption using recent advances in coexistence theory that decompose the outcome of species interactions in patchy landscapes into coexistence-promoting stabilizing niche differences and fitness differences that drive competitive exclusion [@Chesson_2000]. Specifically, I propose to test whether functional trait differences are predictive of the similarity in the functional responses of species to environmental gradients. Another striking observation from nature is that individuals within a species can have vastly different phenotypes. Despite considerable interest, the consequences of this variation on species coexistence remain largely unclear [@Violle_2012; @Turcotte_2016]. For Chapter 2, I aim to investigate how intra-specific variation in functional traits might influence community dynamics beginning from understanding of how trait differences influence species coexistence generally. Over the past two decades, a great deal of research has also highlighted the role of soil microorganisms in shaping terrestrial plant communities [@Bever_2015]. A key insight to emerge from this line of inquiry is that plant species coexistence can be promoted when plants cultivate a soil microbial community that has more negative impacts on plants of the same species than on plants of other species, which is in agreement with traditional coexistence theory [@Chesson_2000; @Bever_2003]. Although these biotic interactions influence plant community dynamcics in nature in conjunction with abiotic factors, their effects are generally considered independently of one another [@van_der_Putten_2016]. For Chapter 3, I aim to demonstrate how the relative contributions of plant-plant competition and plant-microbe interactions can be jointly considered under the established coexistence framework of @Chesson_2008 via a critical review of existing plant-soil feedback literature. This thesis will involve novel applications of recent advances in theoretical ecology to plant community and itegrate the historically disjunct fields of plant competition and plant-microbe interactions."
date: "15 August 2017"
output:
  bookdown::pdf_document2:
    template: ~/grtools/svm-latex-ms.tex
    toc: false
fontfamily: mathpazo
fontsize: 12pt
header-includes: \usepackage{tabularx} \usepackage{float,booktabs,longtable,multirow,color} \usepackage{graphicx,colortbl,xcolor,wrapfig}\usepackage{mathptmx,fixltx2e} \usepackage[font=small,skip=0pt]{caption}
csl: ecology.csl
bibliography: bibliography.bib
---

\setlength{\parindent}{0em}
\setlength{\parskip}{.5em}

\clearpage

## Background

*Overall introduction will go here. I plan to give a broad introduction to species coexistence, describe the benefits of the trait-based approach, and describe how recent advances in microbial ecology have been relevant to plant ecology (1-1.5 pages?)*. 


\clearpage

## \textbf{Chapter 1: How do functional traits relate to species responses to variable environments?}

*Additional collaborators: Andy Kleinhesselink (UCLA); Jonathan Levine (ETH-Zurich); Emily Curd (UCLA/Cal eDNA)*

\bigskip
\indent
A foundational principle of trait-based ecology is that traits are functional to the extent that they influence population demography by directly influencing growth, survival, and reproduction [@Violle_2007]. As variation in demographic properties is at the heart of species coexistence [@Chesson_2000], understanding the links between species traits and their demography is a critical step in using traits to understand coexistence. Although ecologists have long understood that trait differences can reduce the niche overlap between species to promote coexistence [@Macarthur_1967], recent advances have shown that trait differences can also promote competitive exclusion [@Adler_2013; @Kraft_2015; @D_Andrea_2016]. As nature is often strikingly patchy in space and as species often have clumped distributions on landscapes, spatial variation is thought to be an important driver of coexistence in plant communities [@Chesson_2000]. Therefore, traits likely influence species coexistence by their effect on species' demographic variation in space.  

Despite being fundamental to interpreting patterns of trait variation in nature, the links between trait differences and species' demographic responses remain largely unresolved [@Shipley_2016, but see @Adler_2014 and @Wright_2010 for counter-examples] A growing number of studies model the contribution of traits to environmental response by modeling the abundance of species across gradients as a function of their traits [@Laughlin_2014]. However, an important limitation of these studies is that observed abundances are influenced not only by environmental variation but also by biotic variation between sites [@Adler_2013]. Indeed, the most abundant species at a site need not have the traits that confer the highest fitness at that site [@Fox_2012]. Therefore, experimentally isolating the impact of environment while keeping the competitive environment constant is a critical step in building stronger links between functional trait differences and variation in species responses to variable environments. Interpreting these results in light of ecological first principles regarding trait-environment relationships can give generality to the findings.

In addition to the competitive neighborhood, the biotic interactions between plants and microbes may also change across space to influence plant performance. This turnover of soil microbes may have important effects on plant responses to abiotic environmental variation [@Keymer_2017]. For example, @Lau_2012 found that the fruit production of *Brassica rapa* was less impacted by a drought treatment when the plants were grown with a drought-adapted soil microbial community than when grown with a soil microbial community that had not been previously exposed to drought. Therefore, the soil microbial community may function as an additional environmental variable that to date has received relatively little attention from plant ecologists. Recent advances in sequencing technology have made it feasible to characterize variation in soil microbial turnover, and we can begin to assess their role in driving plant environmental preferences.

To address these gaps, I propose to use experimental data to estimate the contribution of trait differences to species-specific demographic responses to environmental variation, including turnover in soil microbial communities, for the first chapter of my dissertation. 

### Chapter 1 hypotheses

\textbf{H\textsubscript{0}}: Species may respond identically to environmental variation, irrespective of functional traits. We may expect this pattern if the environmental gradient spans a single stress gradient axis such that all species perform poorly at some sites and well at others (Figure \ref{fig:ch1-hypotheses}a).  
\textbf{H\textsubscript{1}}: Species may have idiosyncratic responses to the environmental gradient, but the response may be unrelated to species functional traits. There may be potential for the storage effect to promote species coexistence, but the degree of stabilization would be uncorrelated to species' functional similarity (Figure \ref{fig:ch1-hypotheses}a).  
\textbf{H\textsubscript{2}}: Variation in responses of species to environmental gradients may be explained by their functional traits. This pattern might arise if the functional traits are indicative of species' environmental preference along the gradient sampled (Figure \ref{fig:ch1-hypotheses}a).  
\textbf{H\textsubscript{3}}: Species similarity in some traits may correlate with similarity in response to certain environmental variables. For example, species' similarity in drought tolerance may predict their response to an aridity gradient but not to a light gradient, and species similarity in seed size may predict their response to a soil texture gradient but not an aridity gradient (Figure \ref{fig:ch1-hypotheses}b). 


\begin{figure}[h!]
  \includegraphics[width=\textwidth]{img/ch01-hypotheses.pdf}
  \caption{Potential ways in which trait differences may correlate with differences in species' demographic response}
  \label{fig:ch1-hypotheses}
\end{figure}


### Methods

**Demographic variation**: In 2015-2016, we planted seeds of 17 annual plant species (Table \ref{tab:table-species-ch1}) in a setup that allowed me to parameterize a simplified annual plant population growth model. Specifically, the population growth rate of species $j$ at site $x$ is modeled as a function of the germination rate $g_{j,x}$, a global seed-bank survival rate for each species $s_j$, low-density seed production $\lambda_{j,x}$, species response to competitors $r_{j,x}$, and the competitive neighborhood density $\eta_x$: 

$$\frac{N_{t+1,j,x}}{N_{t,j,x}} = (1-g_{j,x})*s_j +\frac{g_{j,x}*\lambda_{j,x}}{1+r_{j,x}*\eta_x}$$
In this framework, environmental response can be parameterized in three ways: germination rate, the per-capita seed production in the absence of competitors, and a composite index that yields the expected population growth rate in the absence of competitors.   


**Trait variation**: A suite of leaf, root, seed, and whole-plant functional traits (*table to be inserted*) were measured for twelve of the annual plant species in this experiment by @Kraft_2015. I have measured the same set of traits for the five remaining species and will additionally measure leaf osmotic potential [@Bartlett_2012], stomatal index, and leaf vein density [@Sack_2013], all of which summarize components of leaf-level drought tolerance.   
\begin{wraptable}{r}{5cm}
\footnotesize
```{r table-species-ch1, echo = F}
library(magrittr); library(knitr)
species <- c("Agoseris heterophylla","Centaurea melitensis" ,"Chaenactis glabriuscula","Hemizonia congesta" ,"Lasthenia californica", "Micropus californicus", 
             "Amsinckia menziesii", 
             "Clarkia bottae","Clarkia purpurea", 
             "Euphorbia peplus", 
             "Medicago polymorpha", "Lotus wrangeliensis", 
             "Bromus madritensis","Hordeum murinum", "Vulpia microstachys", 
             "Salvia columbariae", 
             "Plantago erecta")
families <- c(rep("Asteraceae", 6), "Boraginaceae",rep("Onagraceae", 2), "Euphorbiaceae", rep("Fabaceae",2), rep("Poaceae", 3), "Lamiaceae", "Plantaginaceae")

to_print <- cbind(Family = families, Species = species)

knitr::kable(to_print, caption="Species list for demography study", longtable = TRUE, booktabs = T)
```
\normalsize
\end{wraptable}
**Abiotic environmental variation**: We characterized various soil chemical and physical properties at each of our experimental sites during the 2015-16 growing season. Air temperature at each of the plots was measured with temperature data loggers (iButtons) that were placed inside PVC tubing to shield from solar radiation, and gravimetric water content was measured for twice for each plot - once each in the peak growing season for the early and later phenology species.  

**Biotic environmental variation**: We characterized the variation in soil microbiota among our plots by using metagenomic sequencing to profile the diversity of bacteria, fungi, and small eukaryotes. Soil was pooled from three equally spaced points within each experimental plot into a single sterlize 15 mL tube and stored at -80$^{\circ}$C until further processing. Total DNA was extracted using the MoBio Power Soil Extraction kit and PCRs were conducted using 16s rRNA primers for bacteria, fungal ITS primers, and 18s rRNA primers for small eukaryotes. Pooled samples were sequenced on the Illumina MiSeq platform at the UC-Berkeley sequencing facilities. Sequence data will be used to ordinate sites according to their soil microbial communities in multidimensional space.    

\bigskip 

\textbf{Analyses}: I propose to assess the influence of functional traits on the demographic responses of species to environmental gradients with a generalized linear mixed modeling (GLMM) framework modified from [@Jamil_2012]. This framework, which was developed under the context of the fourth-corner problem [@Legendre_1997] to estimate trait-environment relationships, models species performance across a landscape ($p_{i,j}$) as a function of species traits $z$, environmental properties $x$, and crucially, an interaction term between the two: 
\[  
p_{i,j} \sim \overbrace{a_0}^{species\ effect}+\underbrace{a_1z_j}_{trait\ effect}+\overbrace{b_0x_i}^{environment\ effect}+\underbrace{b_1z_jx_i}_{interaction}+\overbrace{\epsilon_{\alpha j} + \epsilon_{\beta i} + \gamma_i}^{error\ terms}
\]

The species and site effect terms can be thought of as unmeasured species traits and environmental variables, respectively. The contribution of trait differences to interspecific variation in environmental response (i.e. the slope parameter $\beta_1$) can be quantified as a function of the variance explained $\hat\sigma_b$ by a model that includes a trait:environment term and one that lacks this interaction:

$$C_\beta = 1-\frac{\hat\sigma^2_{\beta,\ no\ interaction\ term}}{\hat\sigma^2_{\beta,\ with\ interaction\ term}}$$


### Preliminary results, implications, and further analyses
A large proportion of the species pairs had uncoupled demographic responses- in other words, most species pairs did not respond in the same way to the multivariate changes in environment across sites (Figure \ref{fig-ch1-results}a), suggests against **H\textsubscript{0}**. Thus, variable responses of species to environmental heterogeneity may contribute to their coexistence in this landscape. The environmental variation stems primarily from differences in soil [Ca\textsuperscript{2+}]:[Mg\textsuperscript{2+}] ratio and in soil texture and depth  (Figure \ref{fig-ch1-results}b). \textcolor{BrickRed}{The soil microbial community also varied across our landscape (currently running scripts to get an NMDS plot from this as a first step)}. The species in this experiment differed primarily in leaf economics spectrum traits on the first axis of variation and on whole plant traits on the second axis of variation (Figure \ref{fig-ch1-results}c). Priliminary analyses conducted with simple linear regressions (not in the GLMM framework above) suggest that some trait differences may explain variation in species responses to environment. For example, four out of the five smallest-seeded species in this experiment responded positively to shallower and sandier soils, whereas all other species prefer deeper soils with higher clay content (Figure \ref{fig-ch1-results}d). 


Modeling variation in demographic response to environment in this framework above will address a fundamental gap of trait-based plant ecology: the assumption functional traits have direct and predictive relationships with measurable environmental gradients [@Adler_2013;@Shipley_2016]. Due to strong separation of sites along axes of soil chesmitry and physical structure, root and seed traits that directly link to species' belowground ecological strategies may be more relevant to explaining variation in species response in our system than traits that primarily inform aboveground strategies. Alternatively, physiological traits with a clear link to an environmental axis (e.g. osmotic potential to water availablility) may show stronger links to plant performance than integrative traits like SLA that are related to various environmental axes. An important step will be to conduct a power analysis to assess the extent to which the potentially weak but important influences of individual traits (as might be expected for some of the integrative functional traits measured for this study) can be recovered given the probabalistic nature of many demographic models (Snyder & Ellner, *in prep*). A number of studies have also found phylogenetic structure in plant communities; therefore, incorporating evolutionary non-independence of our samples by using phylogenetically informed GLMMs [@Ives_2011] is another promising next step. I plan to interpret all results in light of ecological first-principles and in the framework of modern coexistence theory to yield generality to these findings. Armed with a more robust understanding of the link between traits and demography, trait-based plant ecology can move towards its potential to be a predictive science in a rapidly changing world [@McGill_2006]



\begin{figure}[h!]
  \includegraphics[width=\textwidth]{img/ch1-fig2full.png}
  \caption{a)Species tended to show decoupled demographic responses (all pairwise correlations in species responses <1. b) Soil chemical and physical properties were the primary axes of variation among sites. c) Species differed primarily in their leaf economics traits along PC axis 1 and primarily on whole-plant traits along PC axis 2. d) Four of the five smallest seeded species had a positive response to Environment axis 2, unlike all of the bigger seeded species. \textbf{This is a low-quality draft figure, will have better one later}}
  \label{fig-ch1-results}
\end{figure}

\clearpage

## Timeline

|**Year**|  **Season** |  **Plan** |
|:-----:|------|--------------------------------------------------------|
| 2017 | Fall-Winter  | Finalize data collection and Analysis for Chapter 1; Literature reviews for Chapter 3   |
| 2018 | Spring-Summer| Analysis and writeup for Chapter 1; Additional data collection for Chapter 2; Literature reviews & modeling for Chapter 3|
| 2018 | Fall-Winter  | Submission for Chapter 1, Analysis for Chapter 2, Modeling & analysis for Chapter 3 |
| 2019 | Spring-Summer| Writeup and Analysis for Chapter 2, Writeup for Chapter 3|
| 2019 | Fall-Winter  | Writeup for Chapter 2; Submission for Chapter 3|
| 2020 | Spring| Submission for Chapter 2|


\clearpage

## References





